tests/template_tests/test_extends.py

```diff
@@ -117,3 +119,16 @@ def test_unique_history_per_loader(self):
         template = engine.get_template('base.html')
         output = template.render(Context({}))
         self.assertEqual(output.strip(), 'loader2 loader1')
+
+    @override_settings(INSTALLED_APPS=[
+        'template_tests.extended',
+        'template_tests.base',
+    ])
+    def test_block_override_in_app_extended_included_template(self):
+        """
+        Ensure that an includable template containing a block, where the
```

Comment:
All tests ensure expected behavior ;-). Our preference is to use test docstrings to state and explain the expected behavior.